File: Class 2/Problem 26/removeDuplicate.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j=0
        i=0
        
        while j < len(nums):
            if nums[j] != nums[i]:
                i+=1
                nums[i] = nums[j]
            j+=1
        
        return i+1

File: Class 2/Problem 26/removeDuplicate.js
let nums = [1, 1, 2];

var removeDuplicate = function (nums) {
  let j = 0;
  let i = 0;

  for (j = 0; j < nums.length; j++) {
    if (nums[j] !== nums[i]) {
      i++;
      nums[i] = nums[j];
    }
  }

  return i + 1;
};

console.log(removeDuplicate(nums));
